save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/ProjectInfo.kt

```diff
@@ -0,0 +1,135 @@
+/**
+ * Function component for project info and edit support
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package org.cqfn.save.frontend.components.basic
+
+import org.cqfn.save.entities.Project
+import org.cqfn.save.frontend.externals.fontawesome.faCheck
+import org.cqfn.save.frontend.externals.fontawesome.faTimesCircle
+import org.cqfn.save.frontend.externals.fontawesome.fontAwesomeIcon
+
+import react.FC
+import react.Props
+import react.dom.html.InputType
+import react.dom.html.ReactHTML.button
```

Comment:
`react.FC` rather than `react.fc` and DOM elements from `react.dom.html.ReactHTML` is a new kotlin-react API, which uses builders without `attrs`